src/main/java/at/hannibal2/skyhanni/data/model/SkyblockStat.kt

```diff
@@ -1,62 +1,161 @@
-    SWING_RANGE("§eⓈ"),
-    SPEED("§f✦"),
-    SEA_CREATURE_CHANCE("§3α"),
-    MAGIC_FIND("§b✯"),
-    PET_LUCK("§d♣"),
-    FISHING_SPEED("§b☂"),
-    BONUS_PEST_CHANCE("§2ൠ"),
-    COMBAT_WISDOM("§3☯"),
-    MINING_WISDOM("§3☯"),
-    FARMING_WISDOM("§3☯"),
-    FORAGING_WISDOM("§3☯"),
-    FISHING_WISDOM("§3☯"),
-    ENCHANTING_WISDOM("§3☯"),
-    ALCHEMY_WISDOM("§3☯"),
-    CARPENTRY_WISDOM("§3☯"),
-    RUNECRAFTING_WISDOM("§3☯"),
-    SOCIAL_WISDOM("§3☯"),
-    TAMING_WISDOM("§3☯"),
-    MINING_SPEED("§6⸕"),
-    BREAKING_POWER("§2Ⓟ"),
-    PRISTINE("§5✧"),
-    FORAGING_FORTUNE("§☘"),
-    FARMING_FORTUNE("§6☘"),
-    MINING_FORTUNE("§6☘"),
-    FEAR("§a☠")
+enum class SkyblockStat(
+    val icon: String,
+    @Language("RegExp") tabListPatternS: String,
+    @Language("RegExp") menuPatternS: String,
+) {
+    DAMAGE("§c❁", "", ""),  // Weapon only
+    HEALTH("§c❤", " Health: §r§c❤(?<value>.*)", " §c❤ Health §f(?<value>.*)"), // TODO get from action bar
+    DEFENSE("§a❈", " Defense: §r§a❈(?<value>.*)", " §a❈ Defense §f(?<value>.*)"), // TODO get from action bar
+    STRENGTH("§c❁", " Strength: §r§c❁(?<value>.*)", " §c❁ Strength §f(?<value>.*)"),
+    INTELLIGENCE("§b✎", " Intelligence: §r§b✎(?<value>.*)", " §b✎ Intelligence §f(?<value>.*)"), // TODO get from action bar
+    CRIT_DAMAGE("§9☠", " Crit Damage: §r§9☠(?<value>.*)", " §9☠ Crit Damage §f(?<value>.*)"),
+    CRIT_CHANCE("§9☣", " Crit Chance: §r§9☣(?<value>.*)", " §9☣ Crit Chance §f(?<value>.*)"),
+    FEROCITY("§c⫽", " Ferocity: §r§c⫽(?<value>.*)", " §c⫽ Ferocity §f(?<value>.*)"),
+    BONUS_ATTACK_SPEED("§e⚔", " Attack Speed: §r§e⚔(?<value>.*)", " §e⚔ Bonus Attack Speed §f(?<value>.*)"),
+    ABILITY_DAMAGE("§c๑", " Ability Damage: §r§c๑(?<value>.*)", " §c๑ Ability Damage §f(?<value>.*)"),
+    HEALTH_REGEN("§c❣", " Health Regen: §r§c❣(?<value>.*)", " §c❣ Health Regen §f(?<value>.*)"),
+    VITALITY("§4♨", " Vitality: §r§4♨(?<value>.*)", " §4♨ Vitality §f(?<value>.*)"),
+    MENDING("§a☄", " Mending: §r§a☄(?<value>.*)", " §a☄ Mending §f(?<value>.*)"),
+    TRUE_DEFENCE("§7❂", " True Defense: §r§f❂(?<value>.*)", " §f❂ True Defense §f(?<value>.*)"),
+    SWING_RANGE("§eⓈ", " Swing Range: §r§eⓈ(?<value>.*)", " §eⓈ Swing Range §f(?<value>.*)"),
+    SPEED("§f✦", " Speed: §r§f✦(?<value>.*)", " §f✦ Speed §f(?<value>.*)"), // TODO add the way sba did get it (be careful with 500+ Speed)
+    SEA_CREATURE_CHANCE("§3α", " Sea Creature Chance: §r§3α(?<value>.*)", " §3α Sea Creature Chance §f(?<value>.*)"),
+    MAGIC_FIND("§b✯", " Magic Find: §r§b✯(?<value>.*)", " §b✯ Magic Find §f(?<value>.*)"),
+    PET_LUCK("§d♣", " Pet Luck: §r§d♣(?<value>.*)", " §d♣ Pet Luck §f(?<value>.*)"),
+    FISHING_SPEED("§b☂", " Fishing Speed: §r§b☂(?<value>.*)", " §b☂ Fishing Speed §f(?<value>.*)"),
+    DOUBLE_HOOK_CHANCE("§9⚓", " Double Hook Chance: §r§9⚓(?<value>.*)", ""),
+    BONUS_PEST_CHANCE("§2ൠ", " Bonus Pest Chance: §r§2ൠ(?<value>.*)", " §2ൠ Bonus Pest Chance §f(?<value>.*)"),
```

Comment:
```suggestion
    BONUS_PEST_CHANCE("§2ൠ", " (?:§r§7§m)?Bonus Pest Chance: (?:§r§2)?ൠ(?<value>.*)", " (?:§7§m|§2)ൠ Bonus Pest Chance (?:§f)?(?<value>.*)"),
```
account for disabled